requirements.txt

```diff
@@ -4,11 +4,11 @@
 
 # Avoid method=restore bug: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4209#issuecomment-1739644328
 # dipy 1.8.x causes conflicts, see https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4319#issuecomment-1864670675
-dipy!=1.6.*,!=1.7.*, !=1.8.*
+dipy!=1.6.*,!=1.7.*
 # PyTorch's Linux distribution is very large due to its GPU support,
 # but we only need that for training models. Our users only need CPU.
 --extra-index-url https://download.pytorch.org/whl/cpu # append_to_freeze
-ivadomed~=2.9.6
+ivadomed @ git+https://github.com/ivadomed/ivadomed@jn/update-pinned-dependencies # append_to_freeze
```

Comment:
Because we've been limited to `nibabel==3.x` for so long, the jump to `5.x` introduced some test failures due to API changes.

I'll try and fix those up so that SCT is compatible with modern `nibabel`.